microsoft-edge/webview2/howto/debug.md

```diff
@@ -120,7 +123,7 @@ In Visual Studio Code, complete the following actions to debug your code.
     :::image-end:::
     
     > [!NOTE]
-    > Because Visual Studio Code does not perform source mapping, ensure you set breakpoints in the same file that WebView2 uses.  If the paths do not match, Visual Studio Code does not pause the running code at the breakpoint.  
+    > Because Visual Studio Code source path mapping requires the URL configuraiton setup your application will recive a command line parameter on startup.  If uneeded you can safely ignore this parameter.  
```

Comment:
receive*, not needed* or unneeded* I think?

Reply:
Spelling fixed.